.github/workflows/ci.yml

```diff
@@ -58,3 +86,27 @@ jobs:
           npm --prefix integration/${{ matrix.suite }} install
           npm --prefix integration/${{ matrix.suite }} install "file:/./$PKG_TARBALL"
           npm --prefix integration/${{ matrix.suite }} test
+
+
+  integration-bun:
+    needs: test
+    runs-on: ubuntu-latest
+
+    env:
+      REPLICATE_API_TOKEN: ${{ secrets.REPLICATE_API_TOKEN }}
+
+    strategy:
+      matrix:
+        bun-version: [1.0.11]
+        suite: [bun]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Bun ${{ matrix.bun-version }}
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: ${{ matrix.bun-version }}
+      - run: |
+          cd integration/${{ matrix.suite }}
```

Comment:
Lets copy the steps from the above jobs to build the packfile and install it, so we're running against equivalent of the published package and not the local one.

```
PKG_TARBALL=$(npm --loglevel error pack)
npm --prefix integration/${{ matrix.suite }} install
npm --prefix integration/${{ matrix.suite }} install "file:/./$PKG_TARBALL"
```